test/python/test_wrapper.py

```diff
@@ -141,6 +143,18 @@ def test_local_execute_and_get_ran_qasm(self):
                 result = execute(self.circuit, 'local_qasm_simulator').result()
                 self.assertIsNotNone(result.get_ran_qasm(self.circuit.name))
 
+    def test_qobj_to_quantum_circuit(self):
```

Comment:
can you add one more test for a qobj containing multiple circuits?

also make it such that at least one of the circuits contains more than one quantum register or more than one classical register. This case will be of interest when we do pure qobj -> circuit translation, because qobj flattens the registers into just 1. so we have to be careful when reconstructing the circuit, and a test like this will allow us to verify correctness later.

Reply:
Done